data/ext/pending/issue-3563.ttl

```diff
@@ -0,0 +1,117 @@
+@prefix : <https://schema.org/> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix xml: <http://www.w3.org/XML/1998/namespace> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+:MemberProgram a rdfs:Class ;
+	rdfs:label "MemberProgram" ;
+    :isPartOf <https://pending.schema.org> ;
+    :source <https://github.com/schemaorg/schemaorg/issues/3563> ;	
+    rdfs:comment "A MemberProgram defines a loyalty (or membership) program that provides its members with certain benefits, for example better pricing, free shipping or returns, or the ability to earn loyalty points. Member programs may have multiple tiers, for example silver and gold members, each with different benefits." ;
+    rdfs:subClassOf :Intangible .
+	
+:MemberProgramTier a rdfs:Class ;
+	rdfs:label "MemberProgramTier" ;
+    :isPartOf <https://pending.schema.org> ;
+    :source <https://github.com/schemaorg/schemaorg/issues/3563> ;	
+    rdfs:comment "A MemberProgramTier specifies a tier under a loyalty (member) program, for example \"gold\"." ;
+    rdfs:subClassOf :Intangible .
+
+:isTierOf a rdf:Property ;
+    rdfs:label "isTierOf" ;
+    :domainIncludes :MemberProgramTier ;
+    :rangeIncludes :MemberProgram ;
+	:inverseOf :hasTiers ;
+    :source <https://github.com/schemaorg/schemaorg/issues/3563> ;
+    rdfs:comment "The member program this tier is a part of." .
+
+:hasTiers a rdf:Property ;
+    rdfs:label "hasTiers" ;
+    :domainIncludes :MemberProgram ;
+    :rangeIncludes :MemberProgramTier ;
+	:inverseOf :isTierOf ;
+    :source <https://github.com/schemaorg/schemaorg/issues/3563> ;
+    rdfs:comment "The tiers of a member program." .
+
+
```

Comment:
Nitpick, consistent number of empty lines.

Reply:
done, also converted tabs to spaces